wsl-pro-service/internal/controlstream/controlstream_test.go

```diff
@@ -217,37 +217,68 @@ func TestConnect(t *testing.T) {
 func TestSend(t *testing.T) {
 	t.Parallel()
 
-	ctx, cancel := context.WithCancel(context.Background())
-	defer cancel()
+	testCases := map[string]struct {
+		changePort string
+
+		wantErr bool
+	}{
+		"Success": {changePort: "FALSE"}, // Cannot use "" as default because we want to test the empty string
```

Comment:
we usually do `empty == "-"`, no?

Reply:
I don't think we have any standard yet in this repo (we defaulted to ""). Let's use "-" then.